### PR TITLE
Upgrade to discord.js 13.3 (Carl's branch)

### PR DESCRIPTION
**Pull request by nickmccurdy**
discord.js is fixing thread support in version 13, so it seems like we'll need to upgrade to it to get the bot working in threads.

- [x] Upgrade discord.js
- [ ] Fix type errors
- [ ] Use [command handling](https://discordjs.guide/creating-your-bot/command-handling.html) (optional but allows support for slash commands)

### CODE REVIEW

**Comment by nickmccurdy** [comment]
> @nickmccurdy I think I have fixed type errors in command.ts

This is still a work in progress on my end, but if you want you can send me a Gist with your `command.ts` and I can try to integrate it with my work.

**Comment by Padmanabh82** [comment]
#120 is my pr sir

**Comment by vcarl** [comment]
@Padmanabh82 we are aware of your PR, but it was not something we asked for and thus it's not our top priority to review. We'll get to it — it's inappropriate to comment on unrelated PRs or issues to link it

**Comment by nickmccurdy** [comment]
Since Discord.js has breaking changes in each minor release, the unpushed code I have locally isn't compatible with these changes, so I'll give this branch to you and continue my work in another branch. I'd still like to see if we can merge changes from both after we make more progress though.

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "eslint-plugin-prettier": "^3.1.2",
     "prettier": "^2.3.2",
     "ts-node-dev": "^1.0.0-pre.44",
-    "typescript": "^3.8.2"
+    "typescript": "^4.5.4"
```
Comment (nickmccurdy) [inline package.json:39]: There are a bunch of new type errors with this TypeScript version. I'm guessing we'd either need to update type dependencies or fix our TS config.

File: src/features/emojiMod.ts [hunk 2]
```diff
@@ -65,32 +65,28 @@ const handleReport = (
...
       reaction.users.remove(member.id);
-      member.send([
-        "Hey there! 👋",
-        "The ⚠️ reaction is reserved for staff usage as part of our moderation system.  If you would like to mark a message as needing moderator attention, you can use react with 👎 instead.",
-        "Thanks!",
-      ]);
+      member.send(`Hey there! 👋
+The ⚠️ reaction is reserved for staff usage as part of our moderation system.  If you would like to mark a message as needing moderator attention, you can use react with 👎 instead.
+Thanks!
+`);
```
Comment (nickmccurdy) [inline src/features/emojiMod.ts:80]: This changes it from 3 messages to 1 message with 3 lines, not sure if that was intentional.

**Comment by nickmccurdy** [comment]
@vcarl I assume you won’t be using this branch, but if you change your mind, feel free to reopen.

**Pull request closed without merging**